reference-implementation/__tests__/json/url-specifiers.json

```diff
@@ -47,6 +54,24 @@
       "expectedResults": {
         "/test": "https://example.com/lib/test2.mjs"
       }
+    },
+    "Non-special vs. special schemes": {
```

Comment:
FYI this section currently needs blink internals APIs on WPT+Chromium, as `data:`, `about:` etc. schemes cannot be intercepted by service workers. I'll create a separate PR to move this section to e.g. `url-specifiers-internal.json`.

Reply:
Or you can split this section in this PR out to a separate file like:

```
{
  "importMap": {
    "imports": {
      "data:text/": "/lib/test-data/",
      "about:text/": "/lib/test-about/",
      "blob:text/": "/lib/test-blob/",
      "blah:text/": "/lib/test-blah/",
      "file:text/": "/lib/test-file/",
      "ftp:text/": "/lib/test-ftp/",
      "ws:text/": "/lib/test-ws/"
    }
  },
  "importMapBaseURL": "https://example.com/app/index.html",
  "baseURL": "https://example.com/js/app.mjs",
  "name": "URL-like specifiers",
  "tests": {
    "Non-special vs. special schemes": {
      "expectedResults": {
        "data:text/javascript,console.log('foo')": "data:text/javascript,console.log('foo')",
        "data:text/": "https://example.com/lib/test-data/",
        "about:text/foo": "about:text/foo",
        "about:text/": "https://example.com/lib/test-about/",
        "blob:text/foo": "blob:text/foo",
        "blob:text/": "https://example.com/lib/test-blob/",
        "blah:text/foo": "blah:text/foo",
        "blah:text/": "https://example.com/lib/test-blah/",
        "ftp:text/foo": "https://example.com/lib/test-ftp/foo",
        "ftp:text/": "https://example.com/lib/test-ftp/",
        "file:text/foo": "https://example.com/lib/test-file/foo",
        "file:text/": "https://example.com/lib/test-file/",
        "ws:text/foo": "https://example.com/lib/test-ws/foo",
        "ws:text/": "https://example.com/lib/test-ws/"
      }
    }
  }
}
```

Reply:
I split it out into a separate file, and included HTTP, HTTPS, and WSS.